lib/wind/utf8.c

```diff
@@ -220,8 +237,8 @@ wind_ucs4utf8(const uint32_t *in, size_t in_len, char *out, size_t *out_len)
             default:
                 break;
 	    }
+	    out += len;
```

Comment:
We already `out += len` above.  The one below that you moved here looks suspicious to me.